### ISSUE TITLE
Not compatible with http 1.0.0

### ISSUE THREAD

**Issue by zakblacki** [martin-braun-net/google_maps_place_picker_mb#56]
Please update 

> Because google_maps_place_picker_mb >=2.0.0-mb.22 depends on http ^0.13.5 and project depends on http ^1.0.0, google_maps_place_picker_mb >=2.0.0-mb.22 is forbidden.
> So, because project depends on google_maps_place_picker_mb ^3.0.2, version solving failed.

**Comment by martin-braun** [comment]
I tried to, but it's blocked by https://github.com/lejard-h/google_maps_webservice/issues/149

Don't really have the time to replace `google_maps_webservice`. Feel free to create a PR on https://github.com/martin-braun/google_maps_place_picker_mb/tree/dart-3.

**Comment by BobbyBakes** [comment]
Is this coming off hold anytime soon.  
There are a lot of packages now that depend on http 1.0 and above. I think there will end up being a lot of folks with dependency issues. 

I just hit one with the latest flutter 3.0 upgrades

**Comment by giorgio79** [comment]
Some peeps already updated google_maps_webservice. Just need to replace in pubspec with the latest at 
https://github.com/hectorhasap/google_maps_webservice

recommend cloning the repo first :)

```
  google_maps_webservice:
    git:
      url: https://github.com/hectorhasap/google_maps_webservice
      ref: main
      publish_to: none
```

**Comment by BobbyBakes** [comment]
This was a nice library to get me through MVP. But ended up dropping it since it lacks the maintenance I needed. 
Thanks for a nice run. 

If anyone is in the same boat with this dependency. Rebuilding from `google_maps_flutter` isn't too much of a major task.

**Comment by martin-braun** [comment]
@BobbyBakes I'm glad it helped you along the way. The thing I just ask myself at this point is, why not simply contributing instead of building your own? Rebuilding is definitely more work than making a PR. I do review PRs and happily merge them if they are clean enough.

The constant version bumps that cause Flutter to complain about version mis-matches are a real down-straw to the ecosystem. There are ways around that, but they are not fulfilling, so I share the frustration when it comes to other packages that are unmaintained. I still think it's the better decision to fork and fix the issues and send a PR upstream. You don't have to wait for me to accept your PR to use your changes right away (see above).

Thanks and have a nice day! :)

**Comment by BobbyBakes** [comment]
> , why not simply contributing instead of building your own
It's a matter of where effort goes innit. This package was a means to an end for a roadmap which stopped at MVP. 
The next version entails a customized map which is more productionized. 

If capacity becomes free I wouldn't mind updating this for the success of others.

**Comment by martin-braun** [comment]
Fair enough.

**Comment by martin-braun** [comment]
Solved in `3.1.2`
